code/index.html

```diff
@@ -2,21 +2,108 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>&#127846 CupcakeMania</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!-- Link for google fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
+    <!-- Header with logo and navbar -->
     <header>
-      <!-- Some kind of logo and navbar -->
+      <a href="#">
+        <img class="menu" src="./Images/menu_icon.svg" alt="menu image">
+      </a>
+      <div class="logo"></div>
+      <nav>
+        <div>
+          <a href="#">ARCHIVE</a>
+          <a href="#">INFO</a>
+        </div>
+        <div>
+          <a href="#">CONTACT</a>
+          <a href="#">MORE</a>
+        </div>
+      </nav>
     </header>
 
+    <!-- BIG NEWS SECTION -->
     <section class="big-news">
-      <!-- The big news content -->
+      <h1>WHAT'S NEW IN THE CUPCAKE UNIVERSE?</h1>
+      <div class="big-text-div">
+        <img src="./Images/pic2_unsplash.jpg" alt="Cupcake image">
+        <div>
+          <h2>Here is some BIG news</h2>
+          <p>
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna 
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation 
+            ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+            Duis aute irure dolor in reprehenderit in voluptate velit 
+            esse cillum dolore eu fugiat nulla pariatur. Excepteur 
+            sint occaecat cupidatat non proident, sunt in culpa qui 
+            officia deserunt mollit anim id est laborum."
+          </p>
+        </div>
+      </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- SMALL NEWS SECTION  -->
+    <section class="small-news-container">
+      <div class="grid-1 grid">
+        <img src="./Images/pic1_unsplash.jpg" alt="Cupcake image">
+        <p>Sprinkles will take your cupcakes to the next level.</p>
+        <br>
```

Comment:
The element `<br>` is not semantic. Try to work around it by adding margin/padding to surrounding elements